safe/gui/tools/wizard/step_kw30_field.py

```diff
@@ -134,16 +138,23 @@ def on_lstFields_itemSelectionChanged(self):
             # Generate description for the field.
             field_type = layer_fields.field(field_name).typeName()
             field_index = layer_fields.indexFromName(field_name)
-            unique_values = self.parent.layer.uniqueValues(field_index)[0:48]
+            unique_values = self.parent.layer.uniqueValues(field_index)
             unique_values_str = [
                 i is not None and unicode(i) or 'NULL'
-                for i in unique_values]
+                for i in unique_values[0:48]]
             unique_values_str = ', '.join(unique_values_str)
             field_descriptions += tr('<b>Field name</b>: {field_name}').format(
                 field_name=field_name)
             field_descriptions += tr(
                 '<br><b>Field type</b>: {field_type}').format(
                 field_type=field_type)
+            if len(unique_values) == feature_count:
```

Comment:
Some providers won't be able to provide the feature count. For instance, the feature count on an OSM file will always return -1, because we can't count features in an OSM file efficiently. So can we skip this block if it's -1? It would not make sense to display `from -1 features`.